docs/build-mvm/how-to/schedule_functions.md

```diff
@@ -1 +1,885 @@
 # Cloudless Scheduler
+
+Tested on:
+
+- the FCLI version: `@fluencelabs/cli/0.13.5 linux-x64 node-v18.19.0`
+- Network env: `local`
+
+**TODO: when all updates are published, update to `kras` and the latest FCLI**
+
+# Create your first Spell: step-by-step
+
+1. Initialize a project with `fluence init -t minimal` 
+    
+    The spell library dependency should be added to the `fluence.yaml` configuration file automatically:
+    
+    ```yaml
+    # FCLI version: 0.13.5
+    #...
+    
+    aquaDependencies:
+      "@fluencelabs/aqua-lib": 0.9.1
+      "@fluencelabs/spell": 0.6.9
+    ```
+    
+2. Initialize a new spell template with `fluence spell new myFirstSpell`. Type `Yes` , or just press `return` , to add the spell to the default deal [link to the page explaining deals].
```

Comment:
1. not sure if we need this kind of details, probably we can assume that user can press enter or type yes cause it's everywhere in CLI and it's kinda treatet like a given
2. if there is no link at least for now probably we can remove this. Or if you find a link please update

Reply:
I expect this links to be filled with other pages in the documentation to make sure that the user know what to do and where to look